alert-policies/nextcloud/CPUUsage.yml

```diff
@@ -1,11 +1,11 @@
-name: Scrape Errors
+name: CPU Usage
 
 description: |+
-  This alert is triggered  when the cache size limit is reached above 4 for at least 5 minutes.
+  This alert is triggered  when the CPU Usage is exceeds 90% for at least 5 minutes.
```

Comment:
```suggestion
  This alert is triggered  when the CPU Usage exceeds 90% for at least 5 minutes.
```

Reply:
Hi @mdumpati ,
As per your comments, I have made changes. Kindly review it.
Thanks...